### DISCUSSION TITLE
A bunch of news bots I'm following all stopped posting around the same time.

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#23294] [Q&A] [answered]
NY times last post 1/28/23 11:15a ET

BBC World news 1/28/23 10:45a ET

BBC breaking news 1/28/23 7:30a ET 

Breaking News bot 1/27/23 7p ET

Was there some mastodon update that broke bots? I got like nobody to follow here. I am really trying to switch away from Twitter but this sucks

**Comment by trwnh** [comment, marked as answer]
you'd have to get in contact with whoever manages those bots

**Reply by ghost** [reply]
I was asking here because it's entirely different companies and yet they all stopped around the same time. Its not like one company that manages them all. That's why I thought it's a mastodon change that broke them.

**Reply by trwnh** [reply]
The bots aren't managed by Mastodon, and they're probably not being managed by those companies either. I can't say for sure because I don't know which accounts you're talking about, but the most likely thing is that some random person set up crossposter bots from their Twitter accounts, and probably Twitter revoked their access to the API. Or the server that hosts the bots went offline.

**Reply by ghost** [reply]
Gotcha. the API thing would make sense. Thanks.

**Reply by ghost** [reply]
I just noticed the bots are all on the same mastodon server (mediastodon.com). Is it possible the server I'm on (mastodon.social) blocked that server?

**Reply by trwnh** [reply]
maybe, but it looks like https://mediastodon.com/ is actually having errors right now -- going to their site shows an error page.
